rust/parse_ast/src/convert_ast/converter.rs

```diff
@@ -769,3 +823,35 @@ pub(crate) fn update_reference_position(buffer: &mut [u8], reference_position: u
   buffer[reference_position..reference_position + 4]
     .copy_from_slice(&insert_position.to_ne_bytes());
 }
+
+pub(crate) fn get_outside_class_span_decorators_info<'a, F>(
+  span: &Span,
+  get_class: F,
```

Comment:
Why is this a function to get the class and not directly an `Option<&'a Class>`?

Reply:
That makes sense!